.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgresql://postgres@db/test_track_development
```

Comment:
I like that idea. We could also just define this environment variable in the docker-compose.yml file since it's the only one